src/Pages/Music/Albums.elm

```diff
@@ -43,27 +42,51 @@ type alias Params =
 type alias Model =
     { album_list : List AlbumObj
     , route : Route
+    , currentButton : Selected
+    , titleToggle : Bool
+    , dateToggle : Bool
     }
 
 
 init : Shared.Model -> Url Params -> ( Model, Cmd Msg )
 init shared url =
-    ( { album_list = shared.album_list, route = url.route }, Cmd.none )
+    ( { album_list = shared.album_list, route = url.route, currentButton = Title, titleToggle = True, dateToggle = False }, Cmd.none )
 
 
 
 -- UPDATE
 
 
 type Msg
-    = ReplaceMe
+    = TitleButtonMsg
+    | DateButtonMsg
+
+
+sbylabelalbum : List AlbumObj -> List AlbumObj
```

Comment:
Please don't shorten stuff, I needed a moment to figure out what `s` should mean. Also capitalize the different words.

Reply:
I will keep that in mind.